attackstyles/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -278,10 +286,17 @@ private void updateWarning(boolean weaponSwitch)
 			{
 				if (warnedSkills.contains(skill))
 				{
-//					if (weaponSwitch)
-//					{
-//						// TODO : chat message to warn players that their weapon switch also caused an unwanted attack style change
-//					}
+					if (weaponSwitch)
+					{
+						var message = String.format("Warning, flagged attack style detected: %s!", skill.getName());
```

Comment:
I wouldn't use `String::format` here. `ChatMessageBuilder` already is a `StringBuilder`, so doing this would be more efficient. I also agree with kyle on this
```suggestion
						chatMessageManager.queue(QueuedMessage.builder()
							.type(ChatMessageType.GAMEMESSAGE)
							.runeLiteFormattedMessage(new ChatMessageBuilder()
								.append(Color.RED, "Warning, flagged attack style detected: ")
								.append(skill.getName())
								.append("!")
								.build())
							.build());
```

Reply:
Good point!